PiwikTracker/Queue.swift

```diff
@@ -1,32 +1,20 @@
 import Foundation
 
-protocol Queue {
-    associatedtype T
-    /// The number of queued items.
-    var itemCount: Int { get }
+public protocol Queue {
```

Comment:
I refactored the queue so taking elements doesn't remove them. This way we don't lose events at an app crash during transmission.